back/src/infra/http/ExpressAdapter.ts

```diff
@@ -12,6 +12,13 @@ export class ExpressAdapter implements Http {
       res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept, Authorization')
       res.header('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE, OPTIONS')
 
+      const bodyParams = Object.keys(req.body)
+      bodyParams.forEach((key) => {
+        if (req.body[key] === 'false') {
+          req.body[key] = false
+        }
+      })
+
       if (!req.headers.authorization && req.method === 'PUT') {
```

Comment:
I believe need to validate the key value, what do you think?
```suggestion
      if (req.headers.authorization !== process.env.SECRET_KEY && req.method === 'PUT') {
```

Reply:
hahah, its very important! Thanks!

Reply:
Now I add a test case for this